stormlog/phases/replay.py

```
@@ -0,0 +1,297 @@
+    if isinstance(event, Mapping):
+        return event.get(field_name, default)
+    return getattr(event, field_name, default)
+
+
+__all__ = [
+    "PhaseBoundaryRecord",
+    "PhaseSpan",
+    "PhaseReplayIndex",
+    "is_phase_boundary_event",
+    "parse_phase_boundary",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)